xcode/Subconscious/Shared/Components/FirstRun/FirstRunProfileView.swift

```diff
@@ -19,12 +19,12 @@ struct FirstRunProfileView: View {
                     ValidatedTextField(
                         placeholder: "nickname",
                         text: Binding(
-                            get: { app.state.nicknameTextField },
+                            get: { app.state.nicknameFormFieldValue },
                             send: app.send,
-                            tag: AppAction.setNicknameTextField
+                            tag: AppAction.setNickname
                         ),
                         caption: "Lowercase letters, numbers and dashes only.",
-                        hasError: !app.state.isNicknameTextFieldValid
+                        hasError: !app.state.isNicknameFormFieldValid
```

Comment:
Nit: consistency, see below. 

Do we prefer `app.state.isNicnameFormField.isValid` or `app.state.isNicknameTextFieldValid`?